testmon/pytest_testmon.py

```diff
@@ -118,10 +116,16 @@ def is_active(config):
 
 def pytest_configure(config):
     if is_active(config):
+        init_testmon_data(config)
         config.pluginmanager.register(TestmonDeselect(config, config.testmon_data),
                                       "TestmonDeselect")
 
 
+def pytest_unconfigure(config):
+    if hasattr(config, 'testmon_data'):
+        config.testmon_data.close_connection()
```

Comment:
Not sure if this is really required, but cannot hurt.
On the other hand, it adds the `close_connection` method.